docker/setup.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/sh
+set -euo pipefail
+
+DATABRICKS_TF_VERSION=$(/app/databricks bundle debug terraform --output json | jq -r .terraform.version)
+DATABRICKS_TF_PROVIDER_VERSION=$(/app/databricks bundle debug terraform --output json | jq -r .terraform.providerVersion)
+
+# Download the terraform binary
+mkdir -p zip
+wget https://releases.hashicorp.com/terraform/${DATABRICKS_TF_VERSION}/terraform_${DATABRICKS_TF_VERSION}_linux_${ARCH}.zip -O zip/terraform.zip
+unzip zip/terraform.zip -d zip/terraform
```

Comment:
Can we confirm the hash of this binary?

Reply:
fixed in https://github.com/databricks/cli/pull/1369